modules/nf-core/blast/blastdbcmd/tests/main.batch.nf.test

```diff
@@ -0,0 +1,47 @@
+nextflow_process {
+
+    name "Test Process BLAST_BLASTDBCMD_BATCH"
+    script "../main.nf"
+    process "BLAST_BLASTDBCMD_BATCH"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "blast"
+    tag "blast/blastdbcmd"
+
+    setup {
+        run("BLAST_MAKEBLASTDB") {
+            script "../../makeblastdb/main.nf"
+            process {
+                """
+                input[0] = [ [id:'test2'], file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
```

Comment:
We have a new syntax now (not yet official published): https://github.com/nf-core/website/blob/fc31c5404460fa5bf4cb4b00f499ad35658bb900/src/content/docs/guidelines/components/modules.md#87-input-data